tests/selenium/test_selenium_logged_in_flow.py

```
@@ -0,0 +1,528 @@
+        )
+        patient = PatientUser.objects.create(user=user, active=True)
+
+        # Create a denial and appeal for this user
+        denial = Denial.objects.create(
+            hashed_email=Denial.get_hashed_email(user.email),
+            denial_text="Test denial",
+            patient_user=patient,
+            procedure="MRI Scan",
+            diagnosis="Chronic Headaches",
+        )
+        appeal = Appeal.objects.create(
```

Ruff (0.14.10)

Local variable `appeal` is assigned to but never used

Remove assignment to unused variable `appeal`

(F841)